sklearn/manifold/_mds.py

```diff
@@ -586,6 +604,20 @@ def fit_transform(self, X, y=None, init=None):
         elif self.dissimilarity == "euclidean":
             self.dissimilarity_matrix_ = euclidean_distances(X)
 
+        # TODO(1.4): Remove
+        self._normalized_stress = self.normalized_stress
```

Comment:
I'm okay with passing this directly into `smacof` and have the function raise the warning.